### PR TITLE
develop: fix paging sql with multiple columns for cluster index table

### PR DESCRIPTION
**Pull request by Mini256**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)

Change the paging SQL example with multiple columns for the cluster index table.

The old example will make a bad full table scan, the new one can use indexes.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v6.4 (TiDB 6.4 versions)
- [x] v6.3 (TiDB 6.3 versions)
- [x] v6.2 (TiDB 6.2 versions)
- [x] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs/pull/10478
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Comment by Oreoxmt** [comment]
@Icemap @winoros PTAL

**Review by Icemap — APPROVED** [review]
LGTM

**Review by Oreoxmt — APPROVED** [review]
LGTM

**Comment by Mini256** [comment]
/hold

Need to change further.

**Comment by Mini256** [comment]
/unhold

@Icemap @winoros PTAL again, thanks!

File: develop/dev-guide-paginate-results.md [hunk 1]
```diff
@@ -306,6 +306,10 @@ GROUP BY page_num
 ORDER BY page_num;
 ```
 
+> **Note:**
```
Comment (Oreoxmt) [inline develop/dev-guide-paginate-results.md:309]:
```suggestion
> **注意：**
```

File: develop/dev-guide-paginate-results.md [hunk 2]
```diff
@@ -306,6 +306,10 @@ GROUP BY page_num
...
+>
+> 该 SQL 会以 `TableFullScan` 方式执行，当数据量较大时，查询速度会变慢，这里可以[使用 TiFlash](/tiflash/tiflash-overview.md#使用-tiflash)进行加速。
```
Comment (Oreoxmt) [inline develop/dev-guide-paginate-results.md:311]:
```suggestion
> 该 SQL 会以全表扫描 (TableFullScan) 方式执行，当数据量较大时，查询速度会变慢，此时可以[使用 TiFlash](/tiflash/tiflash-overview.md#使用-tiflash) 进行加速。
```

**Review by Oreoxmt — APPROVED** [review]
Rest LGTM

**Comment by Mini256** [comment]
@Oreoxmt, thanks for your review!

**Pull request merged**